sensor_msgs/PointCloud2 cloud
---
grasp_generator/GraspConfigList grasp_configs
